main.py

```
@@ -10,6 +10,7 @@
 from fastapi.exceptions import RequestValidationError
 from api.frame.frame_routes import router as frame_router
 from api.diagnosis.diagnosis_routes import router as diagnosis_router
+from api.diagnosis.diagnosis_routes_v2 import router as diagnosis_router_v2
```

Ruff (0.11.9)

Module level import not at top of file

(E402)